__tests__/app.test.js

```diff
@@ -1,18 +1,18 @@
 const endpointsJson = require("../endpoints.json");
 const request = require("supertest");
-const app = require("../app")
+const app = require("../app");
 /* Set up your test imports here */
 const db = require("../db/seeds/seed"); // the file we use seed to enter data into the database
```

Comment:
is this the best naming convention for the seed, and likewise the database connection?